dali/operators/reader/loader/cufile_loader.h

```diff
@@ -38,12 +38,30 @@ namespace dali {
 struct ImageFileWrapperGPU {
   Tensor<GPUBackend> image;
   std::string filename;
-  bool transpose_fortan_order;
-  TensorShape<> shape;
-  TypeInfo type_info;
+  bool fortran_order;
```

Comment:
This field seems out of place. Regular files don't have a concept of axis order (nor do they have shape or type).

Reply:
Where do you suggest that we put it? Another struct dedicated to NumpyReader?

Reply:
I think that would be the cleanest option. Or we could use some generic metadata string which could be parsed but that is more complicated than having a numpy specific struct here I think.